timg/argparser.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 
 from . import meta
+from platfrom import system
```

Comment:
I think there's a typo in `platform` module name.

Reply:
whoops

Reply:
done